components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/base/base_container.html

```diff
@@ -179,6 +183,7 @@
         // jsTree code in ome.tree.js and center panel code in center_plugin.thumbs.js.html uses initially_select
         // instead of browser URL since URL may be /webclient/?path=plate.name-barcode|well.name-A1
         WEBCLIENT.initially_select = [{% for o in initially_select %}"{{ o }}"{% if not forloop.last %},{% endif %}{% endfor %}];
+        WEBCLIENT.initially_open = [{% for o in initially_open %}"{{ o }}"{% if not forloop.last %},{% endif %}{% endfor %}];
```

Comment:
As discussed recently, constructing array this way is danger, see https://github.com/openmicroscopy/openmicroscopy/pull/5018/files